### PR TITLE
ENG-2633 Updated Jetty db migration parameter

### PR DESCRIPTION
**Pull request by cecchisandrone**
Current base images versions are:

```
entando-base-common:6.3.2
entando-wildfly17-base:6.3.7
entando-eap73-clustered-base:6.3.7
```
After merging the PR I will manually build and push to Dockerhub:
```
entando-base-common:6.3.3
entando-wildfly17-base:6.3.8
entando-eap73-clustered-base:6.3.8
```
Once the new images will be available I will create another PR to modify Dockerfiles in [entando-de-app](https://github.com/entando-k8s/entando-de-app)

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-docker-base-images).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-docker-base-images/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by user-54dbb57a — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/entando-docker-base-images/pull/18#pullrequestreview-811064225" title="Approved">w-caffiero-entando</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **entando-jx**

If they are not already assigned, you can assign the PR to them by writing `/assign @entando-jx` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fentando-docker-base-images).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/entando-docker-base-images/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by entando-jx** [comment]
@cecchisandrone: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | 386487b063350beb68084e2b5844070d2e296782 | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
